src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[new Random().nextInt(1000)];
```

Comment:
1000 is a magic number

seems like you missed some of the comments - let's take a look at the hidden conversations

Reply:
add